docs/fluent-conversion.rst

```diff
@@ -0,0 +1,97 @@
+.. This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I've also noticed some oddities in the TOC. I didn't add it because it's a temporary file that's only useful during our transition to the new system. That said this transition period could last for quite a while... I'm happy to add it if you think it'd be more useful there instead of just a link from the l10n doc..